Lombiq.Gulp.Extensions.csproj

```diff
@@ -1,33 +1,31 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Ezt majd azért beszéljétek meg @BenedekFarkas-vel, mert most kérdéses, mit kellene itt.

Reply:
Itt szerintem valami merge hiba lehetett, most néztem végig a history-t, én így kezdtem, de nem került be a netstandard2.0 hozzám miután mergeltem devből

Reply:
Ezt továbbra is így van, és továbbra is a szimpla visszaállítása helyett előtte a @BenedekFarkas -sal való egyeztetést javaslom.

Reply:
Várom a FINI-474 végét, hogy mi legyen végül